django-stubs/core/management/commands/loaddata.pyi

```diff
@@ -6,10 +6,16 @@ from django.core.management.base import BaseCommand
 READ_STDIN: str = ...
 
 class Command(BaseCommand):
+    ignore: bool = ...
+    using: str = ...
+    app_label: str = ...
+    verbosity: int = ...
+    format: str = ...
     missing_args_message: str = ...
+    help: str = ...
```

Comment:
We already have `help` here: https://github.com/typeddjango/django-stubs/blob/master/django-stubs/core/management/base.pyi#L38

Reply:
Thanks. I believe d3ff5415db9306b2d5a766b2167a9d2e1a548ce2 resolves this.